client/src/app/site/modules/global-headbar/components/account-button/account-button.component.ts

```diff
@@ -189,7 +189,22 @@ export class AccountButtonComponent extends BaseUiComponent implements OnInit {
         const userId = this.operator.operatorId;
         if (this._userId !== userId) {
             this._userId = userId;
-            this.user = this.operator.user;
+            this.doUserUpdate();
         }
     }
+
+    private doUserUpdate(): void {
+        if (!this._userId) {
+            this.user = null;
+            return;
+        }
```

Comment:
this check seems unnecessary because in line 191 this.userId is defined as your own id

Reply:
did another change.